app/src/main/java/eu/kanade/tachiyomi/ui/reader/viewer/pager/PagerPageHolder.kt

```diff
@@ -353,7 +353,20 @@ class PagerPageHolder(
     }
 
     private fun mergePages(imageStream: InputStream, imageStream2: InputStream?): InputStream {
-        imageStream2 ?: return imageStream
+        // SY -->
+
+        // Handle adding a center margin to wide images if requested
```

Comment:
mergePages is already a SY specific function, no need to add the SY comment
```suggestion
        // Handle adding a center margin to wide images if requested
```